src/content/getting-started/kubectl-gs/index.md

```diff
@@ -0,0 +1,242 @@
+---
+linkTitle: kubectl-gs introduction
+title: Getting started with kubectl-gs
+description: The Giant Swarm Management API is easily accessible by means of kubectl-gs, a kubectl plugin developed by Giant Swarm. This tutorial guides you throughout some of the most important features of the tool.
+weight: 40
+menu:
+  main:
+    parent: getting-started
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-rainbow
+user_questions:
+  - How can I access the Giant Swarm Management API?
+  - How do I use kubectl gs?
+last_review_date: 2022-02-28
+---
+
+# Getting started with `kubectl-gs`
+
+As better explained in the [reference]({{< relref "/ui-api/kubectl-gs" >}}), kubectl-gs is a CLI and a kubectl plugin (invoked as kubectl gs) for the Giant Swarm Management API. In this guide, you will learn how to use `kubectl-gs` for your daily tasks on the Giant Swarm installation.
```

Comment:
The main headline is now part of the template (identical with `title` in front matter).

```suggestion
---

As better explained in the [reference]({{< relref "/ui-api/kubectl-gs" >}}), kubectl-gs is a CLI and a kubectl plugin (invoked as kubectl gs) for the Giant Swarm Management API. In this guide, you will learn how to use `kubectl-gs` for your daily tasks on the Giant Swarm installation.
```